ci-operator/step-registry/acm/fetch-operator-versions/acm-fetch-operator-versions-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: acm-fetch-operator-versions
+  from: fetch-operator-versions
+  commands: acm-fetch-operator-versions-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  credentials:
```

Comment:
Credentials and env could be removed since they are not used in your script.
To remove everything from line 9 --> line 17